.travis.yml

```diff
@@ -80,10 +80,8 @@ cache:
     #- .tox/
 
 before_install:
-  # If you update these versions, make sure you update the versions in the .st2client-install-check
-  # and the requirements targets in the Makefile to match
-  - pip install --upgrade "pip==20.0.2"
-  - sudo pip install --upgrade "virtualenv==16.6.0"
```

Comment:
FYI pinning `virtualenv` here never really worked properly. In Travis environment even after updating virtualenv to latest:
```
virtualenv --version

16.1.0
```
for py2 env

and

```
virtualenv --version

16.0.0
```
for py3 env

Reply:
It turns out because different virtualenv binaries were used when using `sudo` and without.